docs/examples/exporters/GLTFExporter.html

```diff
@@ -80,7 +80,9 @@ <h3>[method:null parse]( [param:Object input], [param:Function onCompleted], [pa
 			<li>truncateDrawRange - bool. Export just the attributes within the drawRange, if defined, instead of exporting the whole array. Default is true.</li>
 			<li>binary - bool. Export in binary (.glb) format, returning an ArrayBuffer. Default is false.</li>
 			<li>embedImages - bool. Export with images embedded into the glTF asset. Default is true.</li>
-			<li>animations - Array<[page:AnimationClip AnimationClip]>. List of animations to be included in the export.
+			<li>animations - Array<[page:AnimationClip AnimationClip]>. List of animations to be included in the export.</li>
+			<li>forceIndices - bool. Generate indices for non-index geometry and export with it. Default is false.</li>
```

Comment:
Not sure about the wording here, maybe "with them"? Or `Export non-indexed geometry by creating temporary indices for it`? /cc @mrdoob @donmccurdy

Reply:
Thanks, Replaced "it" with "them" so far.